src/component/Text.tsx

```diff
@@ -99,7 +99,7 @@ const calculateWordsByLines = (
   const findLongestLine = (words: Array<Words>): Words =>
     words.reduce((a: Words, b: Words) => (a.width > b.width ? a : b));
 
-  if (!shouldLimitLines) {
+  if (!shouldLimitLines || scaleToFit) {
```

Comment:
again, we should add a unit test for these cases. Where before it did add ellipses, now it doesn't, etc.